src/main/java/kineticrevolution/multiblocks/patterns/PatternFactories/PatternFactory.java

```diff
@@ -0,0 +1,53 @@
+package kineticrevolution.multiblocks.patterns.PatternFactories;
+
+import kineticrevolution.multiblocks.patterns.Pattern;
+import kineticrevolution.multiblocks.patterns.PatternException;
+import kineticrevolution.multiblocks.patterns.PatternFactories.Factories.*;
+
+import java.util.ArrayList;
+
+/**
+ * Created by MrKickkiller on 3/07/2015.
+ */
+public final class PatternFactory {
+    final static ArrayList<IPatternFactory> factories = new ArrayList<>();
```

Comment:
a list and numbered factories doesn't seem the best approach to me, i think an enum would be better here